src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>
```

Comment:
When configuring your Hibernate setup, if you're utilizing the property:
`<property name="hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>`
You need not this line.
```suggestion
      
```

Reply:
It will be used automatically, just a suugestion)